sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_client_async.py

```diff
@@ -62,11 +62,11 @@ def emit(self, record):
a6\xbe\xb7\xf9\x1b\xc0qa3^-\xea\xbc\x82g\x9d\xb9\x7f\xce\x8f\x12,\xdc~(\xd3\x0e\xf7D\xd8MD\xa8VenpU\x08|\t\xf5\xe2nr4\xb1^j`\xb2V\xdd\xae\x83Gf\x93\r\x03\xe6\xc3f*\xa0r\x07\x0e\x1c\xf2\xbe1\xa1;(J"D\xa9\t\xed\x0e\x13\xddu\x9f\xac\xba\x83h\x0bE\x9a\x12\r\'\x0e\xc7\x99+\x9e\x96,\x8e\x10\xcfG\xbdQ\xda\xf0"\x80\xd4\x83<\xc5\xd1\x7f\x9e\xbb\x06\x11\x8a\xf0\xfce-=\xa1ueq\x01\xa1\xe1N\xd7!\xfe\xe8\x99\x13\xd9,\x86o7\xa2\xd8\xa0\x02J\x1a\xaef\x91j&\xf1\xfa\x81+qd\xfehh\xe4\xf4P\x816\x10\xac\xfd\xfbj^M\xa8D\xe5r\xf7/\xb5\xe6\xab\xf5*{;\x11(\xe0A\xb1\x1f\xaeM z\xc3\xc7V\x9b\x07\x9ajZ\x010\xbdbF\xc3I,1md\xf2o\x8b\xbc\xd62\xad1\x8dLh/{\xa50\x9f#?o\xee\xd6k\xa6\xd2\\\xa5;n\x88\x16\xa8;W\xbe8\x81+u\x14\xf1\xd1\xebl\xf0\x1bl\xd5\xd9\xbea\xbb_\xe16xWxOJ2\xc7p\xc1\xd7\xcbRG*^\xb3A\x06\xa9\x03\x8c]vK\x8a\xed\xecp\x9a\x02<u|O\xce\xfe\x83*\x0b\xc6\x04;t\xd8k\xb9\x8b\xe2f\xa3\xeeT}%\xad\'\xee\x05G\xa4\xa1\x88\xcb\xa2\x14z\xf6\x9d>\x1c\\\xbc\xcd\x7ft\xc6\xb1;\xf8]\xe3\x10\x83@-U\xf0\x16*\xc6\xcb\xfa@~\x1do\x13\xa3\xdd\x84\xee\xc4>\xe6ED\x88\xeen A\xb1\xb6\x8c9Ta5\x8ct&\x90\xa3\r\x91\x08\xf0D\xec\x95\xa4\xed\xe7\xef_\xf4p\x92t\x17\x8bn\x93pk\x15\x84\x06t\xff\x94\xdf\xdb\xe2\x8f\xd2\x93\x81g\x96>\xe2\xc4\x1cm H]4*M\x1e>\xa91&N\x8b\x12\x8f\x83\x93\x0c\xe3\xbam/\x9bVy\xach\xd0\t\xf4\xe5Z\xf5\xb3@]#\xa5\xfb\xa5\x8f\x123Q\xdf>6[\xf7\xdd\xfa\xc5$\xd1\x8d\x06S\x86\x1c6 H|\xc3\xe4\x91\t,Vj\x94\xdd\x8d2\xca\xe6\xb3\xad\x99$\xff}\xa4\xa9*\xce5\xfcU\xf4\x96\xe4\x80\xb1\xb0K\x83\xb6,\x10\xc2\xb9\xbcG\xc8\x1a\x1a\x92\xa2\xef\x8f!b{\xf8\xd8\xa7\xb1\xee\xe1\x1e\x97\x11\xf0\xf4\x9a\na\xa3\xa8aXx{\x83.x\xc8\x0f\xd1\x88\xaa\x19g\xb22&\xfdx)\xb1s\xb3G\xfc<F\xd2\xc7\xbeQ\xa5\xf5D\xb5\x00\xfeE\x9c\x88o.\xe8\x9a\x9c\x11\x9cr\xef\xc3N \x8fI\x16\xcbS\x11\xf3\x84\x14 \xbe\xf2\xa0\x92H:\x02\xfa)\xc8\x1c\x8b\x839e\xa5]p\xd0+\xd0\'f\x8c\xd2\x9f8\x1e\xc3"\xf0\xd2\xc4)Y\x8c\xbfx\xcc\xe6\xca\xc6\xc9\xc4\xc5\xa9\xb1\xb3\x9fw\x1c\xb9\n&A\x1d\x86\xc9)>f\xaa/\xf4\xed\xdc\x86\x8b\xab\x1b\x89\xb0\\\xfb\x88\x8d\x8a=\x1c\x03\xf4\xdff|OR}\xcfr\xbc\x98\x8b\xe7t7\xb1\x9a\xbd\x8fz*\xd5\xb5\x18k[\x9bn\xfd>\xe0X\xf7K\x088\x16cs\xed0\x8d\xee\xcc-o`Q\x9f\x04\xcb\x87\xd0\x917p`Y\xa4O \xecf\t\xea,\x97Q\xb3\xa9k\xfe\xb0\xdf\xc6l\xa7\x89\xf0\xdc\xa5\x9aMM\xcd\xa5^&\r\r\x90\xad\x93\xf6\x19l\xb5W\xe6u\x19\xbe^N\x15\xdfJF\xfcP\xfa\xefnuO[\xe4,!\xcf\xe82r\xb2\x81\x06\xb1\xf8\x92\xd8\xe3>\xff\x1d\xe8\x9c\xcb\xeb\x07\xa4\xfe\x85\xf5&\xb6\xfcJrt;w\x80\xcc~a\xc2\xf7\x08\xffa\x99\x14,\xb6\x08k\xeb\x1d\xf4\xf6\xeb\xd5M\xa6\'\xa9j\xc9k\x0fP\xce\xd7,3\x1f\xcc\xddW:\x02\x03\xbb7\x18\xc54Q\xe50\xe2c(=\x85\x89,0\xd7\xaaJ+\x02\xb3/\xce\x0e\xd3\n\xc2N\xf7\x9bo\xb4\x19\xdb\xc8\x19\xb3\xd0}^\xdc\x80`D\x073}Z\x1d\x81-klf\xadt\xed\x97\xab3M\x97\x99\x11f\xab\xf3\x17\xd16\x8e\xb2eW\x851%0#\x06\t*\x86H\x86\xf7\r\x01\t\x151\x16\x04\x14\xecw|\xc23\rW\xad`\xbf0\x18\xe0\xb7\xc2\x1b\x15\xb0\x8b\xf2010!0\t\x06\x05+\x0e\x03\x02\x1a\x05\x00\x04\x14.\xa0\x17\x15I\xbb\xf1\x94;\x08$na\xc4\xb7;\xda\xaa\xdbQ\x04\x08\x9a\xba\x98\xec*6\xda`\x02\x02\x08\x00'
```

Comment:
Suggest using the same certs in both modules and having one import them from the other